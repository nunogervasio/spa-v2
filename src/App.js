import React from "react";
import AboutUs from "./AboutUs/aboutUs";
import Feature from "./Feature/feature";
import Footer from "./Footer/footer";
import "./App.css";
import Header from "./Header/header";

function App() {
  return (
    <div>
      <Header />
      <AboutUs />
      <Feature />
      <Feature swap={true} />
      <Footer />
    </div>
  );
}

export default App;
