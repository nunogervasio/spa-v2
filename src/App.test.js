import React from "react";
import { shallow } from "enzyme";
import ReactDOM from "react-dom";
import App from "./App";
import toJson from "enzyme-to-json";

it("renders properly", () => {
  const app = shallow(<App />);
  expect(toJson(app)).toMatchSnapshot();
});

// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });
