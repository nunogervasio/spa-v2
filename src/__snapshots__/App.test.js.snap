// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div>
  <Header />
  <AboutUs />
  <Feature />
  <Feature
    swap={true}
  />
  <Footer />
</div>
`;
